spec/lib/map/security_token/service_spec.rb

```diff
@@ -12,40 +12,20 @@
     let(:log_prefix) { '[MAP][SecurityToken][Service]' }
 
     shared_examples 'STS token request' do
-      context 'when an issue occurs with the client request' do
-        let(:expected_error) { Common::Client::Errors::ClientError }
-        let(:expected_error_message) do
-          "#{log_prefix} token failed, client error, status: #{status}, description: #{description}," \
-            " application: #{application}, icn: #{icn}"
-        end
-        let(:status) { 'some-status' }
-        let(:description) { 'some-description' }
-        let(:raised_error) { Common::Client::Errors::ClientError.new(nil, status, { error_description: description }) }
-
-        before do
-          allow_any_instance_of(described_class).to receive(:perform).and_raise(raised_error)
+      context 'when response is not successful with a 401 error' do
```

Comment:
Is 401 the only HTTP status we care about wrt SUS API errors?

Reply:
I think so, honestly with these services I don't see much value in parsing all their possible errors, just the ones we might expect based on our interactions (for example they do have a specific 'grant type missing' error, but there would be no reason for us to ever expect to see that error).

The only other error that might be relevant is an 'ICN not found' error. However, both vets-api and Sign Up Service will source likely ICN from the same place, so we shouldn't ever see that error (and their response for it is messed up so will probably create a Sentry event. In this case I'm happy for that to happen if it ever does)